### PR TITLE
demos: Thumper demo

### PR DESCRIPTION
**Pull request by amx-piap**

### CODE REVIEW

**Comment by amx-piap** [comment]
@pablogs9 I have a strange error in the CI:

**Comment by pablogs9** [comment]
These links seem to be ok, but let me rerun the CI

**Comment by pablogs9** [comment]
Done, I just removed Twitter from CI. Are you ok with that @ralph-lange ?

**Review by ralph-lange — APPROVED** [review]
Yes, I'm fine with that.

File: _docs/tutorials/demos/thumper_demo/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Thumper a 6 wheels robot
```
Comment (jamoralp) [inline _docs/tutorials/demos/thumper_demo/index.md:2]:
```suggestion
title: Thumper, a six-wheeled robot
```

**Review by jamoralp — APPROVED** [review]

**Pull request merged**